endoreg_db/serializers/video/video_processing_history.py

```
@@ -0,0 +1,153 @@
+        if not isinstance(value, dict):
+            raise serializers.ValidationError("config must be a dictionary")
+        
+        operation = self.initial_data.get('operation')
+        
+        # Validate masking config
+        if operation == VideoProcessingHistory.OPERATION_MASKING:
+            required_fields = ['mask_type']
+            if 'mask_type' not in value:
+                raise serializers.ValidationError(
+                    f"Masking config must include: {', '.join(required_fields)}"
+                )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)